index.js

```diff
@@ -1,40 +1,50 @@
 if (process.env.NODE_ENV !== "production") {
       require('dotenv').config();
-  }
+}
 
 const express = require('express');
 const app = express();
 const mongoose = require('mongoose');
 const methodOverride = require('method-override');
+const studentRoutes = require('./routes/student')
```

Comment:
Route requires should be separate from module requires
route require are beeter to be written after the connection of mongo established :)